doc/parameters.md

```diff
@@ -186,6 +186,30 @@ before the client is throttled by adding latency.
 When set to _true_, APIcast will log the response code of the response returned by the API backend in 3scale. In some plans this information can later be consulted from the 3scale admin portal.
 Find more information about the Response Codes feature on the [3scale support site](https://access.redhat.com/documentation/en-us/red_hat_3scale/2.saas/html/analytics/response-codes-tracking).
 
+### `APICAST_SERVICES_FILTER_BY_URL`
+**Value:** a regular expression
+**Default:**: .*
+**Example:** *.example.com
+
+Used to filter the service configured in the 3scale API Manager, the filter
+matches with the public base URL. The services that don't match the filter will
+be discarded. If the regular expression cannot be compiled no services will be
+loaded. 
+
+Note: If a service does not match, but is included in the
+`APICAST_SERVICES_LIST` the service will not be discarded
+
+Example:
+
+Regexp Filter: http:\/\/^test.*.com
+Service 1: backend endpoint http://test.foo.com
+Service 2: backend endpoint http://prod.foo.com
+Service 3: backend endpoint http://test.bar.com
```

Comment:
We have a check in Travis that verifies that all the links in the doc files are valid.
Unfortunately it also picks these ones and 3 and 4 are not valid :/